### ISSUE TITLE
DoctrineORMModule is now 1.0.0 but DoctrineDataFixtureModule composer requires ~0.7

### ISSUE THREAD

**Issue by catadoctari** [Hounddog/DoctrineDataFixtureModule#51]
Hi,

the new DoctrineORMModule is released since some time now to version 1.0.0 (https://github.com/doctrine/DoctrineORMModule/releases/tag/1.0.0).

I will really like to use your module with the DoctrineORMModule 1.0.0 but the composer Required ~0.7 is set a little bit low:
- hounddog/doctrine-data-fixture-module 0.0.4 requires doctrine/doctrine-orm-module ~0.7 -> satisfiable by doctrine/doctrine-orm-module[0.10.0, 0.11.0, 0.7.0, 0.8.0, 0.8.0-beta1, 0.8.0-beta2, 0.8.0-beta3, 0.8.0-beta4, 0.9.0, 0.9.1, 0.9.2] but these conflict with your requirements or minimum-stability.

THX!

**Comment by mihailgoosbg** [comment]
+1
Please upgrade module so that it is ZF3 compatible

**Comment by dkorsak** [comment]
+1

**Comment by dkorsak** [comment]
ping

**Comment by mikemix** [comment]
I guess this module is dead :(

**Comment by christopher-francisco** [comment]
RIP

**Comment by roelvanduijnhoven** [comment]
Forking it and changing the version *does* seem to work. So I wonder if there is any reason behind not updating? 

If there is no actual reason we might fork it and try to fix some problems.

**Comment by mikemix** [comment]
ZendFramework is dead. Simply switch to Symfony

**Comment by roelvanduijnhoven** [comment]
Yeah sure seems so. However: migrating an *existing* application running on ZF to Symfony does not seem like a good idea ;).

**Comment by roelvanduijnhoven** [comment]
Btw; I do spot some problems with the update. I'll dive into them and will report back.
